terraform/helm/chaos/templates/_helpers.tpl

```diff
@@ -0,0 +1,63 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
```

Comment:
dumb question: Is there some way we can make these helper files more DRY? It looks like very generated code to me that is repeated in many helm charts in our repo.